### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by XhmikosR**
* @fullcalendar/bootstrap             ^4.4.0  →    ^4.4.2
* @fullcalendar/core                  ^4.4.0  →    ^4.4.2
* @fullcalendar/daygrid               ^4.4.0  →    ^4.4.2
* @fullcalendar/interaction           ^4.4.0  →    ^4.4.2
* @fullcalendar/timegrid              ^4.4.0  →    ^4.4.2
* @ttskch/select2-bootstrap4-theme     ^1.3.2  →   ^1.4.0
* bootstrap4-duallistbox              ^4.0.1  →    ^4.0.2
* datatables.net                    ^1.10.20  →  ^1.10.21
* datatables.net-autofill-bs4         ^2.3.4  →    ^2.3.5
* datatables.net-bs4                ^1.10.20  →  ^1.10.21
* datatables.net-buttons-bs4          ^1.6.1  →    ^1.6.2
* datatables.net-fixedcolumns-bs4     ^3.3.0  →    ^3.3.1
* datatables.net-fixedheader-bs4      ^3.1.6  →    ^3.1.7
* datatables.net-keytable-bs4         ^2.5.1  →    ^2.5.2
* datatables.net-responsive-bs4       ^2.2.3  →    ^2.2.5
* datatables.net-rowgroup-bs4         ^1.1.1  →    ^1.1.2
* datatables.net-rowreorder-bs4       ^1.2.6  →    ^1.2.7
* datatables.net-scroller-bs4         ^2.0.1  →    ^2.0.2
* filterizr                           ^2.2.3  →    ^2.2.4
* flot                                ^4.2.0  →    ^4.2.1
* jquery-validation                  ^1.19.1  →   ^1.19.2
* jszip                               ^3.3.0  →    ^3.4.0
* overlayscrollbars                  ^1.11.0  →   ^1.12.0
* summernote                         ^0.8.16  →   ^0.8.18
* sweetalert2                        ^9.10.8  →   ^9.14.0

All are semver safe versions. I tested most of those, but the more people test this, the better. Also, if there are known issues, let me know and I'll downgrade any deps as needed :)

So basically I kept these dependencies back:

* bootstrap
* bootstrap-switch
* moment (could probably be updated later)

Requires #2747.

### CODE REVIEW

**Comment by ferbinu** [comment]
also please update bootstrap to v4.5
Thank you

**Comment by REJack** [comment]
@XhmikosR I would also update bootstrap to v4.5, I see no problems with this 😄

**Comment by XhmikosR** [comment]
Nah, there are known regressions.

**Comment by XhmikosR** [comment]
We'll release a 4.5.1 in the next weeks.

**Comment by REJack** [comment]
Ok then I merge this now 😄
